README.md

```diff
@@ -2,7 +2,7 @@
 
 Install and consume 3rd-party client-side libraries with ease.
 
-[![Build status](https://ci.appveyor.com/api/projects/status/vc2ixijbk1ak780e?svg=true)](https://ci.appveyor.com/project/madskristensen/libraryinstaller)
+[![Build status](https://ci.appveyor.com/api/projects/status/vc2ixijbk1ak780e?svg=true)](https://ci.appveyor.com/project/madskristensen/LibraryManager)
```

Comment:
I am assuming the entire URL will change anyways once the new build system is in place.